lua/null-ls/info.lua

```diff
@@ -65,29 +78,82 @@ end
 M.show_window = function()
     local client = require("null-ls.client").get_client()
     local bufnr = api.nvim_get_current_buf()
+    local ft = api.nvim_buf_get_option(bufnr, "filetype")
     if not client or not lsp.buf_is_attached(bufnr, client.id) then
         log:warn("failed to get info: buffer is not attached")
         return
     end
 
+    local create_method_info = function(method)
+        local name = methods.readable[method]
+        local active_sources = {}
+        local supported_sources = {}
+
+        for _, source in ipairs(sources.get_available(ft, method) or {}) do
```

Comment:
```suggestion
        for _, source in ipairs(sources.get_available(ft, method)) do
```

This will always return a table.